udata_front/theme/gouvfr/templates/macros/forms.html

```diff
@@ -7,13 +7,14 @@
 {% elif f.type == 'SubmitField' %}
 <button type="submit" class="fr-btn">{{ submit }}</button>
 {% elif f.type == 'BooleanField' %}
-<div class="fr-checkbox-group">
-    {{ f() }}
-    <label class="fr-label" for="{{ f.id }}">
-        {{ f.label.text }}
-    </label>
-</div>
+{{ render_boolean_field(f) }}
```

Comment:
Did you take this update from Flask Security forms?

Reply:
no I made a func for it:
```
{% macro render_boolean_field(f) %}
<div class="fr-checkbox-group">
    {{ f() }}
    <label class="fr-label" for="{{ f.id }}">
        {{ f.label.text }}
    </label>
</div>
{% endmacro %}
```
I did this so i can reorganize the login form after the update